src/hotspot/share/opto/vectornode.cpp

```diff
@@ -917,7 +985,7 @@ Node* LoadVectorMaskedNode::Ideal(PhaseGVN* phase, bool can_reshape) {
       }
     }
   }
-  return NULL;
+  return LoadNode::Ideal(phase, can_reshape);
```

Comment:
Should this call `LoadVectorNode::Ideal`?
I understand you did optimization because `vector_needs_partial_operations` is false for `LoadVectorMaskedNode` in aarch64 case. But what if it is different on some other (not current) platform?

Reply:
Right, calling `LoadVectorNode::Ideal()` is better. I will change this later. Thanks.